src/components/Execution/FullAggregation.jsx

```diff
@@ -54,19 +58,19 @@ const FullAggregation = ({ nodes, webhooks }) => {
     ]
 
     let contributors = [
-      { ...nodes[3], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[4], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[5], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[6], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[7], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[8], level: 2, nbChild: 0, parents: aggregators },
-      { ...nodes[9], level: 2, nbChild: 0, parents: aggregators }
+      { ...data[3], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[4], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[5], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[6], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[7], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[8], level: 2, nbChild: 0, parents: aggregators },
+      { ...data[9], level: 2, nbChild: 0, parents: aggregators }
```

Comment:
This looks very static, I suppose this won't work with a dynamic tree?

Reply:
Removed in later commits